dali/base/dautils.cpp

```diff
@@ -1991,8 +1992,8 @@ IRemoteConnection *getElementsPaged( IElementsPager *elementsPager,
     if (checkConn && !elem->conn)
         return NULL;
     unsigned n;
-    if (total)
-        *total = elem->totalres.ordinality();
+    if (returned)
+        *returned = elem->totalres.ordinality();
```

Comment:
Sure but the new total should be equivalent to the old/sole total, which was all matches - postfiltering.. at the moment it doesn't discount postfilters.

It's a pity to have a 'returned' total + a real total + a filled page.
It would perhaps be clearer if the 'returned' total is only used for paging, to return the number of pages instead (but I wouldn't make a major effort to change it now).